src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
I think we actually don't need to exclude the hosts and unixcoreclrloader when you have removed the dependency of the hosts on coreclr. As @rahku said, although the cmake tree is parsed, the hosts are not being built. My original question stemmed from my not realizing that we are not building these. 
In the existing CMakeLists.txt files, we don't exclude targets from the build for CLR_CROSS_COMPONENTS_BUILD unless the exclusion is necessary. I can actually see that I was too eager to exclude the mscoree, mscorpe and mscorrc about two weeks ago. The issue I had could have been fixed by just a local change in the src/dlls/mscoree/coreclr/CMakeLists.txt to exclude the inject_debug_resources target for CLR_CROSS_COMPONENTS_BUILD.
I have just verified locally that it works too. Could you please augment your change to make the mscoree, mscorpe and mscorrc unconditional, revert the hosts and unixcoreclr change and instead of that wrap `add_custom_target(inject_debug_resources.....` in the src/dlls/mscoree/coreclr/CMakeLists.txt in `if(NOT CLR_CROSS_COMPONENTS_BUILD)` ?